crates/aiken/src/cmd/blueprint/policy.rs

```diff
@@ -2,7 +2,7 @@ use crate::with_project;
 use aiken_lang::ast::Tracing;
 use std::path::PathBuf;
 
-/// Compute a validator's address.
```

Comment:
There was also this typo which was showing up in the -h that I figured I'd fix while I was here